.doc_gen/metadata/s3_metadata.yaml

```diff
@@ -3705,3 +3711,17 @@ s3_Scenario_DoesBucketExist:
                 - s3.java2.does-bucket-exist-main
   services:
     s3: {GetBucketAcl}
+    
+s3_CreatePresignedPost:
+  languages:

```

Comment:
This needs a title, a title abbreviation, synopsis, and category, and then it will show up correctly here and in the README. See the scenarios/basics above it for an example.

Reply:
You're right, my mistake. It gets it from the parent element.